perfzero/lib/benchmark.py

```diff
@@ -69,12 +71,43 @@ def _setup(self):
     self.benchmark_execution_time['checkout_repository'] = (
         time.time() - start_time)
 
+    # Start cloud TPU.
+    if self.config.tpu_parameters is not None:
+      start_time = time.time()
+      utils.install_tpu_tool()
+
+      parameters = json.loads(self.config.tpu_parameters)
+      random_id = ''.join([
```

Comment:
Better to make this self.config.tpu_parameters.create_tpu_id and pass it from the environment? (that way if the perfzero script crashes, there can be something else that can cleanup the tpu here)

Reply:
Thanks for the suggestion. That is a good point. Changed to allow the user to specify the TPU name as part of the tpu_parameters flag.